### ISSUE TITLE
Reproducing poses from paper on "HotGestures"

### ISSUE THREAD

**Issue by Utopiah** [AdaRoseCannon/handy-work#24]
In https://www.cam.ac.uk/research/news/machine-learning-gives-users-superhuman-ability-to-open-and-control-tools-in-virtual-reality and the [HotGestures: Complementing Command Selection and Use with Delimiter-Free Gesture-Based Shortcuts in Virtual Reality](https://ieeexplore.ieee.org/document/10269004) from IEEE Transactions on Visualization and Computer Graphics (2023) by Zhaomou Song, John J Dudley and Per Ola Kristensson they describe series of "shortcuts" as hand poses. Could be interesting to try to reproduce it.

**Comment by AdaRoseCannon** [comment]
This seems like a great idea, you can use the library to dump poses, I can't actively work on this project at the moment but a PR would be great.

**Comment by Utopiah** [comment]
In the ReadMe you mention `additional poses are welcome` but is it documented somewhere how to do get a `.handpose` file?

**Comment by vincentfretin** [comment]
Yes it's in the README, you need to call `dumpHands()` from the `handy-work` module. From the code, it calls generatePose and triggers a download of a untitled.handpose file.
